pipelines/workflows/cityofroswell/assets.py

```diff
@@ -0,0 +1,45 @@
+import dagster as dg
+from dagster_gcp.gcs import GCSResource
+from pipelines.utils.resources import SensorThingsResource
+from .partitions import file_partitions
+
+# ------------------------------------------------------------------------------
+# Asset: get_csv_from_gcs
+# ------------------------------------------------------------------------------
+
+# TODO: 5/23/2025 - Implement FROST API to load parition into SensorThings
+# TODO: 5/23/2025 - Implement dagster sensor to watch for new files
+# TODO: 5/23/2025 - Update definitions at the workflow level to include all resouces and sensors
+# TODO: 5/23/2025 - Move any reusable code to utils
+
+@dg.asset(partitions_def=file_partitions)
+def get_csv_from_gcs(
+    context,
+    gcs_roswell: GCSResource
+    ):
+    
+    client = gcs_roswell.get_client()
+
+    bucket = client.bucket("roswellbubbler_dev") 
+
+    blob = bucket.blob(context.partition_key)
+
+    context.log.info("Fetching blobs from gs://%s/", bucket.name)
+    context.log.info("Found blob: %s", blob.name)
+
+    content = blob.download_as_text()
+    context.add_output_metadata({"gcs_generation": blob.generation})
+
+    return content
+
+# ------------------------------------------------------------------------------
+
+@dg.asset(partitions_def=file_partitions, deps=[get_csv_from_gcs])
+def load_csv_to_sensorthings(
+    context,
+    get_csv_from_gcs,
```

Comment:
`get_csv_from_gcs` is in your dependencies, and isn't called in the function. I'm not familiar with how Dagster works, but do you need it to be an argument for this function?